### ISSUE TITLE
GapRollForward: Allow for larger test size than training size

### ISSUE THREAD

**Issue by jeiglsperger** [WenjieZ/TSCV#51]
I have a time series y with a sequence length of 1800. Each time stamp is equivalent to 2 seconds, meaning the whole time series covers a time span of 3600 seconds or 1 hour.
I want to use the first 5 minutes/ 300 seconds/150 time stamps to predict the remaining time series.
Therefore, I want to do a 3-fold time-series cross-validation during the training of the forecasting model. This means that in each split, I take the first 50 time stamps to train the model, and the next 1650 time stamps to validate the model. In the third fold, I then use 150 time stamps (5 minutes) and predict the remaining 1650 time stamps in my series.
When creating the splitter with
```
splitter = GapRollForward(min_train_size=int(seq_len/3)+1, max_train_size=int(seq_len/3)+1, gap_size=0,
                          min_test_size=int(1800-seq_len),                                             
                          max_test_size=int(1800-seq_len)                                              
                          )                                                                            
split = splitter.split(y[0, :])   
```                                                                     
After looping through the split with
```
for train_index, test_index in split
    train_indexes.append(train_index
    test_indexes.append(test_index) 
```
I get only one split for the train and test set each, ranging from 0-50 and 51-1699, respectively.
3 splits would be possible, I, therefore, wonder why it is not possible to have multiple splits when the test set is larger than the training set?
